README.md

```diff
@@ -84,7 +87,7 @@ order = await client.create_gridpool_order(
     delivery_period=delivery_period,
     order_type=OrderType.LIMIT,
     side=MarketSide.BUY,
-    price=quantity,
+    price=price,
```

Comment:
Oooop! How was that working or passing tests?

Reply:
This is the readme file and it is not tested.
We need to write examples in the code as comments that can be tested by the tooling we have already in place and somehow reference them in the readme file. This will help to provide correct documentation, which is important for the clients